### ISSUE TITLE
WiFi not found IntelAX101

### ISSUE THREAD

**Issue by babolicous** [sebanc/brunch#2007]
Hi, thanks for the great work. I used the Brunch r120  stable and volteer 120. Installation to usb went fine. On signing in, wifi shows to be inactivated. When i try to activate it still shows no wifi and as soon as i leave the menu it shows to be deactivated again. It is an intelAX101 Wifi Card and a 12th Gen Intel(R) Core(TM) i3-1215U 1.20 GHz. I already tried to enable "iwlwifi_backport" in brunch configuration menu but it is not listed. 
Any ideas? Thanks a lot in advance.

**Comment by mikefromnj** [comment]
I am having a very similar issue. Did you ever find a fix?

**Comment by babolicous** [comment]
No, so far no solution to the Problem.Am 23.06.2024 04:58 schrieb mikefromnj ***@***.***>:
I am having a very similar issue. Did you ever find a fix?

**Comment by OurCodeBase** [comment]
I have also the similar issues, Anyone solve it please
